src/script/components/RichTextEditor/plugins/EmojiPickerPlugin/EmojiPickerPlugin.tsx

```diff
@@ -120,7 +120,7 @@ export function EmojiPickerPlugin({openStateRef}: Props) {
       return null;
     }
 
-    return checkForEmojis(info.textContent);
+    return checkForEmojis(text);
```

Comment:
The different between the `text` passed to the function and the `info.textContent` is that:
- `text` contains the text up to the cursor (so removes what is after the cursor)
- `info.textContent` is the entire content of the input bar